cicd/forgeops-tests/tests/smoke/ds/test_ds.py

```diff
@@ -8,32 +8,36 @@
 Test are sorted automatically so that's why it's needed to keep test_0[1,2,3]_ naming.
 """
 # Lib imports
-import unittest
+import pytest
 from requests import get, put, delete
-import logging
 
 # Framework imports
-from config.ProductConfig import DSConfig
+from ProductConfig import DSConfig
+from utils import logger, rest
 
 
-class DSSmoke(unittest.TestCase):
+class TestDS(object):
     dscfg = DSConfig()
 
     @classmethod
-    def tearDownClass(self):
-        """DSSmoke suite teardown"""
-        self.dscfg.stop_ds_port_forward(instance_nb=0)
-        self.dscfg.stop_ds_port_forward(instance_nb=1)
+    def setup_class(cls):
+        """DSSmoke suite setup"""
```

Comment:
Comment is out of date.
Is the comment (and similar elsewhere) even necessary?

Reply:
All these messages are docstrings and appear in the report (need to do modification to have them in the html one but we have them in allure for sure) as decription for the testcase.
Can be also used by pydoc to generate a doc for the tests.